sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryOptions.java

```diff
@@ -109,6 +109,20 @@
 
   void setNumStorageWriteApiStreamAppendClients(Integer value);
 
+  @Description(
+      "The max number of messages inflight that we expect each connection will have when connection pool usage is enabled for storage writes.")
+  @Default.Long(1000)
+  Long getStorageWriteMaxInflightRequests();
+
+  void setStorageWriteMaxInflightRequests(Long value);
+
+  @Description(
+      "The max size in bytes for inflight messages that we expect each connection will have when connection pool usage is enabled for storage writes.")
```

Comment:
Nit: this takes effect whether connection pooling is enabled or not.

Reply:
corrected.